api/src/main/resources/messages.properties

```diff
@@ -228,5 +228,27 @@ adminui.app.configureMetadata.label=Configure Metadata
 adminui.configuremetadata.adminGroups.description=Extension point for the groupings on the Configure Metadata page
 adminui.configuremetadata.adminLinks.description=Extension point for the links on the Configure Metadata page
 
+adminui.editRole.label=Edit Role
```

Comment:
Can you update your branch to have the latest code in master please? That way you get rid of these and keep only the new ones